[2024-10-03T16:50:21.929+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T16:50:21.954+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T05:16:45.265780+00:00 [queued]>
[2024-10-03T16:50:21.964+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T05:16:45.265780+00:00 [queued]>
[2024-10-03T16:50:21.964+0530] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-10-03T16:50:21.989+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-03 05:16:45.265780+00:00
[2024-10-03T16:50:21.994+0530] {standard_task_runner.py:72} INFO - Started process 16794 to run task
[2024-10-03T16:50:21.998+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_ingestion_dag', 'validate_data', 'manual__2024-10-03T05:16:45.265780+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmp89vt5d14']
[2024-10-03T16:50:21.999+0530] {standard_task_runner.py:105} INFO - Job 43: Subtask validate_data
[2024-10-03T16:50:22.047+0530] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T05:16:45.265780+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-03T16:50:22.110+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T05:16:45.265780+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T05:16:45.265780+00:00'
[2024-10-03T16:50:22.111+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T16:50:22.177+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-03T16:50:22.180+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-03T16:50:22.275+0530] {validator.py:1467} INFO - 	28 expectation(s) included in expectation_suite.
[2024-10-03T16:50:22.546+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T16:50:22.546+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T16:50:22.547+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:00, 2331.46it/s]
[2024-10-03T16:50:22.583+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:01, 54.04it/s]  
[2024-10-03T16:50:22.584+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:00, 79.83it/s]
[2024-10-03T16:50:22.619+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 41.10it/s]
[2024-10-03T16:50:22.628+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 208.87it/s]
[2024-10-03T16:50:22.657+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 153.17it/s]
[2024-10-03T16:50:22.657+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 153.17it/s]
[2024-10-03T16:50:22.683+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 153.17it/s]
[2024-10-03T16:50:22.694+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 153.17it/s]
[2024-10-03T16:50:22.694+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 153.17it/s]
[2024-10-03T16:50:22.695+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 491.87it/s]
[2024-10-03T16:50:23.069+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-03T16:50:23.634+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T16:50:23.640+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T16:50:23.641+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=validate_data, run_id=manual__2024-10-03T05:16:45.265780+00:00, execution_date=20241003T051645, start_date=20241003T112021, end_date=20241003T112023
[2024-10-03T16:50:23.697+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T16:50:23.712+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T16:50:23.726+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
